secure/key-manager/src/tests.rs

```diff
@@ -497,15 +485,15 @@ fn verify_manual_rotation_on_chain<T: DiemInterface>(mut node: Node<T>) {
     let new_privkey = Ed25519PrivateKey::generate(&mut rng);
     let new_pubkey = new_privkey.public_key();
     // Increment time to 5min + 1sec, so that the rotation passes through rate limits
-    node.time.increment_by(301);
+    node.time.advance(Duration::from_secs(301));
```

Comment:
Similarly, since this is for mock only, maybe `advance_secs()` might be more user friendly as a convenience function :)